ch03tests/05Mocks.ipynb

```diff
@@ -4,54 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>Please, don't number sections. We can get that done if wanted in the post-processing and it will require lest update.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UCL/rsd-engineeringcourse/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>